apps/web/components/UserInfoMenu.tsx

```diff
@@ -16,10 +16,15 @@ import { useState } from "react";
 
 import { Icon, ICON_NAME } from "@p4b/ui/components/Icon";
 import { useOrganization } from "@/lib/api/users";
+import { useTranslation } from "@/i18n/client";
+import { usePathname } from "next/navigation";
 
 export default function UserInfoMenu() {
   const { data: session } = useSession();
   const theme = useTheme();
+  const pathname = usePathname();
+  const { t } = useTranslation(pathname.split("/")[1], "dashboard");
+
```

Comment:
There is no queryParam like in pages here but we have to find a better way to get the activeLanguage. Using `pathname.split("/")[1]`  is a bit hacky.